### ISSUE TITLE
Request for district Gifhorn, Germany

### ISSUE THREAD

**Issue by Simon0Harms** [mampfes/hacs_waste_collection_schedule#1136]
Hi, would be great if you could support the District Germany
https://www.abfallkalender-gifhorn.de

**Comment by Murdock79** [comment]
I'd be interested in that too. Not Gifhorn, but Helmstedt district. At Alexa, there is the page of www.mankei.eu.

**Comment by panteLx** [comment]
Hi!

You can do that at your own by creating a ICS File. You probably have to upload the custom file to your own webhost because they do not store them on their servers.

**Comment by riker09** [comment]
Yes, unfortunately you have to roll your own ICS, they don't provide a service. The file is obtained after POSTing some data to their webserver, then you can download the ICS file that is suitable for your town.
